### PR TITLE
`Feat` | Blacklist for mutation span kinds

### PR DESCRIPTION
**Pull request by Andreas02-dev**
Check whether the location of the mutation is in a `Using` or in a `Comment`, if so then the mutation location will not be shown correctly and we should fail the run.

Related to #1196

### CODE REVIEW

File: integrationtest/ValidationProject/ValidateStrykerResults.cs [hunk 1]
```diff
@@ -11,6 +15,25 @@ namespace IntegrationTests
 {
     public class ValidateStrykerResults
     {
+        private readonly ReadOnlyCollection<SyntaxKind> _blacklistedSyntaxKindsForMutating =
```
Comment (richardwerkman) [inline integrationtest/ValidationProject/ValidateStrykerResults.cs:18]: Please add whitelisting for nodes inside of a MethodDeclarationSyntax or PropertyDeclarationSyntax

Comment (Andreas02-dev) [reply integrationtest/ValidationProject/ValidateStrykerResults.cs:18]: Will do, we should also add SyntaxKind.ConstructorDeclaration and SyntaxKind.FieldDeclaration to the whitelist.

File: integrationtest/ValidationProject/ValidateStrykerResults.cs [hunk 2]
```diff
@@ -107,10 +139,31 @@ public void SolutionRun()
 
             var report = JsonConvert.DeserializeObject<JsonReport>(strykerRunOutput);
 
+            CheckMutationKindsValidity(report);
```
Comment (richardwerkman) [inline integrationtest/ValidationProject/ValidateStrykerResults.cs:142]: Make this method part of the CheckReportMutantCounts and rename it to CheckReportMutants

**Comment by Andreas02-dev** [comment]
@richardwerkman Is there a reason why `CheckReportTestCounts` isn't called in the `NetFullFramework` test?

**Comment by richardwerkman** [comment]
Congrats on your first contribution! :)

**Pull request merged**